pkg/multitenant/tenantcostmodel/ecpu_model.go

```diff
@@ -196,11 +212,11 @@ func (m *EstimatedCPUModel) BatchCost(
 		// Add cost for the batch.
 		writeCPU = m.lookupCost(m.WriteBatchCost.RatePerNode, m.WriteBatchCost.CPUPerBatch, ratePerNode)
 
-		// Add cost for additional requests in the batch, beyond the first.
-		if bi.WriteCount > 1 {
+		// Add cost for additional requests in the batch, beyond the second.
```

Comment:
nit: can you include a comment justifying why we don't count the second write request? It seems kind of arbitrary without the context that it is related to the implicit commit.

Reply:
Done.